test/CreateTest.php

```diff
@@ -1,18 +1,26 @@
 <?php
 
-class CreateTest extends PHPUnit_Framework_TestCase
+use PHPUnit\Framework\TestCase;
+
+class TarantoolExt extends Tarantool {
+		public $property = 42;
+}
+
+final class CreateTest extends TestCase
 {
+		use TestCaseCompat;
+
 		protected static $port, $tm;
```

Comment:
nit: `private`

Reply:
It seems it worth to change all protected fields to private in the commit, where I added final to the test case classes, right?

Reply:
Or remove both `final` and `protected/private` changes to reduce the commit.

Reply:
Okay, removed `final`. It is not related to any commit in the patchset. I borrowed it from PR #140 w/o re-check whether it is related. My bad.

I'll update the branch with this change at next force-push.